Configurations/unix-Makefile.tmpl

```diff
@@ -689,16 +690,39 @@ tar:
 	 git ls-tree -r --name-only --full-tree HEAD \
          | grep -v '^fuzz/corpora' \
 	 | while read F; do \
-	       mkdir -p $$TMPDIR/$$DISTDIR/`dirname $$F`; \
-	       cp $$F $$TMPDIR/$$DISTDIR/$$F; \
+	       PARENT_DIR=`dirname $$F`; \
+	       mkdir -p $$TMPDIR/$$DISTDIR/$$PARENT_DIR; \
+	       # do not copy empty directories \
+	       IS_DIR=`ls -Al $$PARENT_DIR | grep ^d | grep $$F | wc -l`; \
```

Comment:
Thanks for your quick review comments. I will work on it.  I am trying to setup travis CI system with the same arguments to see why build is failing by this patch https://github.com/openssl/openssl/pull/4114/commits/f9632a60291c9c3d3eac692744311c6a322c3890